index.yaml

```diff
@@ -218,7 +259,7 @@ entries:
       catalog.cattle.io/release-name: fleet-agent
     apiVersion: v2
     appVersion: 0.3.9
-    created: "2022-03-28T14:25:40.763319-07:00"
```

Comment:
created timestamp changes shouldn't exist? I believe this is because you made changes to 100.0.3, removed those changes, and then added them back, which resulted in the old digest but a newer creation timestamp.

Reply:
removed the timestamp update by deleting these changes and re-running make charts